kustomize/commands/localize/localize.go

```diff
@@ -62,6 +68,36 @@ kustomize localize https://github.com/kubernetes-sigs/kustomize//api/krusty/test
 			if err != nil {
 				return errors.Wrap(err)
 			}
+
+			if !f.noVerify {
```

Comment:
that's a good idea! It'll make the code easier to read

Reply:
actually I think that extracting [this code](https://github.com/kubernetes-sigs/kustomize/pull/5544/files#diff-7fd9415482e70f383078565d412ea831671f0d450b47baefcb0f40dcd9faba31R73-R79) into its own function would be better since it repeats. what do you think?

Reply:
Agreed! I think that's a good idea.